src/screens/Messages/List/ChatListItem.tsx

```diff
@@ -236,11 +252,8 @@ function ChatListItemReady({
               (hovered || pressed || focused) && t.atoms.bg_contrast_25,
               t.atoms.border_contrast_low,
             ]}>
-            <UserAvatar
-              avatar={profile.avatar}
-              size={52}
-              moderation={moderation.ui('avatar')}
-            />
+            {/* Avatar goes here */}
+            <View style={{width: 52, height: 52}} />
```

Comment:
Do we need this? Why was the `<Link>` above absolute instead of just having the `PreviewableUserAvatar` here?

Reply:
You're not allowed to nest anchor tags or touchables, so we have to absolutely position it